### DISCUSSION TITLE
Allow AppData directory to be customized by user to facilitate cross-device syncing

### DISCUSSION THREAD

**Discussion by ByALongShot** [Chatterino/chatterino2#4205] [Ideas]
Ideally I’d like to sync things like my highlights, commands, and possibly splits across multiple devices.  Rather than cloud-syncing being built in, I could achieve this more easily if I could point the app towards a specific directory to use for its AppData (i.e. one that is configured separately to sync across devices).

This could potentially lead to some complications around config values like “Custom streamlink path” across systems, but would work as a first step versus the manual process I’m doing now.

For Linux/Mac I can likely work around this via symlinks, but Windows seems like more of a challenge without this, though, I could be wrong.

**Comment by Nerixyz** [comment]
> For Linux/Mac I can likely work around this via symlinks, but Windows seems like more of a challenge without this, though, I could be wrong.

You can use symlinks on Windows as well. For folders, these can be across drives. I did [`mklink`](https://learn.microsoft.com/windows-server/administration/windows-commands/mklink) `/J C:\Users\Johannes\AppData\Roaming\Chatterino2\Settings D:\FunnyFernFolder` where the settings are in `D:\FunnyFernFolder`. Now the settings are saved there.

But I agree that Chatterino should provide something to change the location out of the box. This shouldn't be through a command-line switch, but rather through something similar to the `modes` file.

**Comment by wuast94** [comment]
+1 on this, at least an export and import of general settings would be helpful

**Reply by pajlada** [reply]
Simple exporting & importing can already be achieved by copying and pasting the relevant files from and to your [Chatterino folder](https://wiki.chatterino.com/Settings/#where-is-my-chatterino-folder-located) when Chatterino is closed
